backend/database/migrations/1669168792-create-booking.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('booking', {
+      eventId: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        references: {
+          model: 'event',
+          key: 'id'
+        }
+      },
+      userId: {
+        type: Sequelize.STRING,
+        primaryKey: true,

```

Comment:
allowNull: false.

Reply:
Primary key is never null so it's implicitly done.